### ISSUE TITLE
Please set the $presenter property to your presenter path.

### ISSUE THREAD

**Issue by Skintillion** [caffeinated/presenters#4]
Having this error, I am trying a very basic example just to test.

```
Laravel 5.1
```

```
ErrorException in PresentableTrait.php line 24:
Please set the $presenter property to your presenter path. (View: pathtoview/view.php)
```

Model

``` php
namespace App\Models\Main;

use Caffeinated\Presenter\Traits\PresentableTrait;

use Illuminate\Database\Eloquent\Model;

class Phone extends Model
{
    use PresentableTrait;

    protected $presenter = 'App\Presenters\Phone';

    protected $table = 'd_m_phone';
}
```

Presenter

``` php
namespace App\Presenters;

use Caffeinated\Presenter\Presenter;

class Phone extends Presenter 
{

    public function phoneDisplay()
    {
        return $this->number;
    }

}
```

In the view

```
{{ $phone->present()->phoneDisplay }}
```

Any ideas why it's showing that error? I checked for typos everywhere.

**Comment by gvidal** [comment]
I have exactly the same problem, Were you able to fix it?

Thanks!

**Comment by joseferrao** [comment]
I have the same problem! As you corrected?

**Comment by ggwebdev** [comment]
Me too...

**Comment by joseferrao** [comment]
Gabriel, consegui resolver o problema. Um amigo deu uma olhada e são os namespace. O meu namespace da classe para o presenter de User estava errado. Outra coisa que percebi foi que a pasta do Presenter deveria estar como Presenter, no meu caso estava Presenters.

**Comment by ggwebdev** [comment]
Show de bola José! Agora tá funcionando legal! Grato!

**Comment by ggwebdev** [comment]
Opa, valeu José! Show de bola agora.

**Comment by trevnetmedia** [comment]
Sam here =(

**Comment by webberwang** [comment]
@Skintillion @trevnetmedia @ggwebdev I found this thread when I didn't namespace it correctly. The above looks correct so it's probably the version before, I just tested with 0.2.1 & it's working.

**Comment by ryan-eurecab** [comment]
I had the same problem. I fixed it using the php 5.5 class name resolver ::class

`protected $presenter = App\Presenters\Phone::class;`
